packages/cli/src/commands/new/templatehook.ts

```diff
@@ -0,0 +1,27 @@
+import { z } from 'zod';
+
+import { ProjectBuilder } from '../../project';
+
+type TemplateHook = (builder: ProjectBuilder) => Promise<void>;
+
+export const templateHooks: Record<string, TemplateHook> = {
+    'mcpagent': async (builder) => {
+        const readline = await import('node:readline/promises');
+        const rl = readline.createInterface({ input: process.stdin, output: process.stdout });
+
+        const serverName = await rl.question('Enter MCP server name: ');
+        const mcpUrl = await rl.question('Enter MCP server URL: ');
```

Comment:
It'd be great if the template could provide these, and it would go through all the questions